README.md

```diff
@@ -90,14 +90,14 @@ You can also browse the full [API reference documentation](#api-reference).
 
 ## Configuration Reference
 
-There are multiple ways you can configure this library to perform authentication into your application. You can create a new `plist` file with shared values or directly pass your configuration into the sign in method directly.
+Before start using this SDK, you have to configure it. To configure SDK create `OktaAuthConfig` object and assign it to  `OktaAuth.configuration`. There are multiple ways to create configuration object: you can provide either plist file either create configuration with dictionary. 
```

Comment:
Since most of this content is duplicated below, we can condense it to:

```suggestion
Before using this SDK, you'll need to configure it. Simply create an `OktaAuthConfig` object using one of the methods below.
```

Reply:
Done.